common/core/desktop/src/debug.hpp

```diff
@@ -0,0 +1,74 @@
+/*
+ * Keyman is copyright (C) SIL International. MIT License.
+ *
+ * Keyman Core - Debug class definition
+ */
+
+#pragma once
+
+#include <cassert>
+#include <vector>
+
+#include <keyman/keyboardprocessor.h>
+#include <keyman/keyboardprocessor_debug.h>
+
+namespace km {
+namespace kbp
+{
+
+using debug_item = km_kbp_state_debug_item;
+
+class debug_items : public std::vector<debug_item>
+{
+private:
+  bool _is_enabled;
+public:
+  template<typename... Args>
```

Comment:
Can we put this on one line, or indent the second line?

```suggestion
  template<typename... Args> debug_items(Args&&... args);
```